editorial_material/why_livecoms.tex

```diff
@@ -73,16 +73,17 @@ \section{Preprints, perpetual/living reviews, and overlay journals provide conte
 With preprints, authors can post their articles in advance formal publication, gaining readers, drawing community attention, getting citations, and even receiving feedback and constructive criticism that authors use make these articles stronger before their formal publication.
 Thus, preprints provide a glimpse of a path forward, as they are updatable documents which are current and have real value.
 Posting of preprints is also typically free, lowering the entry barrier. 
-However, in much of CoMS (Physics being a potential exception) they do not yet provide the same degree of academic credit as a peer-reviewed publication. 
+However, in much of computational molecular science (some areas of physics being a potential exception) preprints do not yet provide the same degree of academic credit as a peer-reviewed publication. 
 
 Another glimpse is provided by the concept of ``perpetual'' or ``living'' reviews [ref Zuckerman/Mobley and other references therein]; these begin as traditional review papers but the authors keep them current by releasing new versions regularly which incorporate new developments and/or community feedback.
 These provide the potential that an author who stays in a given field could maintain an authoritative, current document which always reflect the state of the field and the author(s)' views. 
-One of the present authors (DLM) is currently experimenting with just such a review [ref], and it has already expanded significantly to incorporate new developments, even gaining an additional author.
+Two of the present authors (DLM) are currently experimenting with just such a review [ref], and it has already expanded significantly to incorporate new developments, even gaining an additional author.
```

Comment:
I don't understand who the "two of the present authors (DLM)" are; who's the other one and what's the other review?